docs/concepts.rst

```diff
@@ -1,128 +1,130 @@
 
-CumulusCI scopes many of its activities to the project, so you'll always run your CumulusCI commands inside your repository directory. Each project gets its own keychain, which we'll talk about in detail below. Each project has its own set of scratch orgs, which CumulusCI namespaces in the Salesforce CLI keychain to prevent collisions. And each project is linked to a single GitHub repository, so you can always be confident of the scope of the actions you're taking.
-
-CumulusCI knows which project you are working on by the current working directory of your shell. Make sure to change directories inside your project before you run project-specific commands.
-
-In order to be used as a CumulusCI project, a directory must both be a Git repository and contain a ``cumulusci.yml`` configuration file. We'll show you how to get set up with a new or existing CumulusCI project in the (TODO link) Get Started section.
+It's important to note that a project may not even contain a package; for example, it can be built to deliver functionality to records, or test data for QA. A project may constitute the entirety of a product made for release, or may be one of multiple projects banded together to make a product. Basically, whatever feature in your product that needs specialized Salesforce metadata and CumulusCI automation in order to be built and deployed, *especially* packages (we cannot stress this enough!), requires its own unique project. 
 
 Project Structure
 -----------------
 
-A CumulusCI project represents an application or a major component of an application on the Salesforce platform. Each project corresponds with a single managed package, or with an org-based implementation. In this documentation, we'll often speak about a package; if you're building a project that isn't packaged, please take this to refer to an unmanaged package within which your implementation is deployed.
+Project Directory
+^^^^^^^^^^^^^^^^^
 
-Together with the core package, a CumulusCI project encompasses a variety of unpackaged application components, automation, tests, and other support material. We'll quickly survey the primary components of a CumulusCI project here. Each component is covered in more depth later in this guide.
+The root directory of your CumulusCI project. Because each project is linked to a single GitHub repository, CumulusCI knows which project you are working on by the current working directory of your shell. 
 
-Automation Configuration (``cumulusci.yml``)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. tip:: Avoid headaches by making sure you're in the correct repository for your project before running project-specific commands; otherwise, your project will produce an error. (This tends to be one of the biggest troubleshooting issues that developers run into with CumulusCI. Check for this first and you might save yourself an extra trip to this guide.).
```

Comment:
```suggestion
.. tip:: Avoid headaches by making sure you're in the correct repository for your project before running project-specific commands. Otherwise, errors can happen in your project. This issue tends to be one of the biggest CumulusCI troubleshooting issues. Check your repo first and you might save yourself an extra trip to this guide.
```
Try to avoid future tense and avoid omitting a subject, like "This tends to . . ." Specify the subject, as suggested. I like the parenthetical thought but I like it better as a non-parenthetical. ;-)

Reply:
Clean and succinct. Love it. 

That said, I prefer "project will produce an error" because that's _literally_ what will happen. The system will kick back an error, not just that an error can occur because you've saved this or that to the wrong folder.

Reply:
Find a way to say that without using future tense and I'm sold. :)